src/components/HomePage/HomePage.styles.ts

```diff
@@ -1,6 +1,7 @@
 import styled from "styled-components"
 
 export const HomePage = styled.div`
+  font-family: "Roboto", sans-serif;
```

Comment:
I would reather move font-family to global styles

Reply:
Done. Please, review changes

Reply:
We have global styles already, just move it to `src/index.styles.ts` pls